modules/k8s/local/k3d.sh

```diff
@@ -82,6 +82,8 @@ create() {
   helm install stakater stakater/reloader
 
   make k8s/create-ns
+
+  make k8s/create-imagepull-secret
 }
```

Comment:
I think these need to change too, right?

Reply:
good spot

Reply:
fixed